src/Helius.ts

```diff
@@ -345,13 +344,27 @@ export class Helius {
         } catch (err: any | AxiosError) {
             if (axios.isAxiosError(err)) {
                 throw new Error(
-                    `error during getMintlist: ${
-                        err.response?.data.error || err
+                    `error during getMintlist: ${err.response?.data.error || err
                     }`
                 );
             } else {
                 throw new Error(`error during getMintlist: ${err}`);
             }
         }
     }
+
+    /**
+    * Returns the current transactions per second (TPS) rate — including voting transactions.
+     *
+    * @returns {Promise<number>} A promise that resolves to the current TPS rate.
+    * @throws {Error} If there was an error calling the `getRecentPerformanceSamples` method.
+    */
+    async getCurrentTPS(): Promise<number> {
+        try {
+            const samples = await this.rpcClient.getRecentPerformanceSamples(1)
+            return samples[0].numTransactions / samples[0].samplePeriodSecs
```

Comment:
Does this array always have an element and do we need to check the others?

Can probably handle that error more gracefully, otherwise will will just give some undefined error.